test/multiverse/suites/active_record_pg/active_record_test.rb

```diff
@@ -591,8 +591,11 @@ def adapter
     # ActiveRecord::Base.configurations[NewRelic::Control.instance.env]['adapter']
     #       ActiveRecord::Base.configs_for(env_name: NewRelic::Control.instance.env)['adapter']
     #       ActiveRecord::Base.configs_for(env_name: RAILS_ENV)['adapter']
-    # require 'pry'; binding.pry
     # Order.configurations[RAILS_ENV]['adapter']
+    if ENV['ENABLE_PRY']
+      require 'pry'
+      binding.pry
+    end
```

Comment:
I don't think we want to preserve this `pry` reference. My guess is that a PR was accidentally merged with some commented-out, debugging-related code.

Reply:
Ok i removed the `pry` reference from the file